openshift/Dockerfile.shopping

```diff
@@ -0,0 +1,12 @@
+FROM ubi8/nodejs-12
+
+USER 0
+RUN mkdir -p /home/node/app
+COPY base/packages/shopping /home/node/app
+RUN chown -R 1001:0 /home/node/app
+
+USER 1001
+WORKDIR /home/node/app
+ENV HOST=0.0.0.0 PORT=3000
+EXPOSE ${PORT}
+CMD [ "node", "." ]
```

Comment:
Same comment on --max-old-space-size

Reply:
I do not think you have to hard code --max-old-space-size in the image. If you want to specify --max-old-space-size, I think you can do it using the NODE_OPTIONS environment variable.

Reply:
Thanks, its actually possible. You will change your CMD command to the below:

RUN node --max-old-space-size=79000 .

You can even do more in the case of this project where we are using `dotenv` by passing the dotenv bootstrapping command like below

RUN node -r dotenv/config --max-old-space-size=1024 .

With the above being said, I think you need to update the last line to accommodate `dotenv`

Reply:
Last paragraph in my new README.md shows how to easily set environment variables in OpenShift.
The application appears to run fine without setting max-old-space-size. Why do you want to set it?
If you have some specific use case where it needs to be set you can do by setting environment variable NODE_OPTIONS to --max-old-space-size=79000 or whatever. I do not understand why you want it set when running in OpenShift, when it is not done otherwise!?

Reply:
Thanks this is for supporting bootstrapping dotenv. You can see more information on this: https://www.npmjs.com/package/dotenv